### PR TITLE
chore: migrate to cli-table3

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: src/cli/console.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const Promise = require('bluebird');
 const _ = require('lodash');
 const chalk = require('chalk');
-const Table = require('cli-table2');
+const Table = require('cli-table5');
```
Comment (jonasdk) [inline src/cli/console.js]: Shouldn't this be a 3 instead of a 5? Seems to be the reason that Travis fails.

Comment (DanielRuf) [reply src/cli/console.js]: Ah thanks, fixed it.

**Comment by DanielRuf** [comment]
Bump.

**Comment by dkhuntrods** [comment]
Hi @DanielRuf, would you mind updating this PR against master? We've updated Travis to Node LTS versions so it would be good to make sure that CI can pass in the newer versions. Thank you!

**Comment by DanielRuf** [comment]
> Hi @DanielRuf, would you mind updating this PR against master?

Done.

**Comment by dkhuntrods** [comment]
Resolved by #466

**Pull request closed without merging**